ocpmodels/datasets/trajectory_lmdb.py

```diff
@@ -38,6 +39,13 @@ def __init__(self, config, transform=None):
 
         world_size = distutils.get_world_size()
         rank = distutils.get_rank()
+
+        # Set seed.
+        seed = torch.initial_seed() % 2 ** 31 + rank
```

Comment:
Why not `(torch.initial_seed() + rank) % 2**31`?
That way we know the seed will be < 2**31 for sure.